.github/workflows/deploy.yml

```diff
@@ -35,15 +37,29 @@ jobs:
         with:
           repository: 'aws-samples/amazon-chime-sdk'
```

Comment:
I'm assuming this is going to change now that we're putting the demo in the integration folder? Otherwise this won't work for either main or 2x

Reply:
Since we use submodule now this is no longer a concern.